blits.sql

```diff
@@ -45,9 +45,11 @@ game_status varchar(10)
 wager_amt int not null,
 game_type varchar (9)
 	constraint check_type check(game_type in ('Blackjack', 'Coin Flip', 'Slots')) ENFORCED,
-winner varchar(10) not null,
+game_starter varchar(10) not null,
```

Comment:
change to `game_starter varchar(15) not null` otherwise there will be an error because it references to player_id (15 characters)